tests/integration_tests/functional/test_shut_down.py

```diff
@@ -28,7 +28,7 @@ def test_reboot(test_microvm_with_ssh, network_config):
     metrics_fifo_path = os.path.join(test_microvm.path, 'metrics_fifo')
     metrics_fifo = log_tools.Fifo(metrics_fifo_path)
     response = test_microvm.metrics.put(
-        metrics_fifo=test_microvm.create_jailed_resource(metrics_fifo.path)
+        metrics_path=test_microvm.create_jailed_resource(metrics_fifo.path)
```

Comment:
nit: This should probably be in the next commit.

Reply:
True :ok_hand:.